types/postcss-color-gray/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "private": true,
+    "exports": {
+        ".": {
+            "types": {
+                "import": "./index.es.d.ts",
+                "default": "./index.d.ts"
+            }
+        }
+    },
```

Comment:
This needs to match the upstream package exactly, and this is not what upstream says; importing this package from ESM will not hit `index.es.js` because `"module": "index.es.js"` only affects bundlers.

I think you're better off not including the `es.d.ts` file at all.